### PR TITLE
feat: run performance simulation on merge queue

### PR DESCRIPTION
**Pull request by stephhuynh18**
The work flow should be in rust-ceramic... but i made one here to match the hermetic tests one.

### CODE REVIEW

**Review by nathanielc — APPROVED** [review]
Looks Great!

**Review by smrz2001 — APPROVED** [review]
🥳

**Review by dav1do — APPROVED** [review]
Nice work! A couple thoughts but nothing that needs to be addressed.

File: hermetic/src/cli/tester.rs [hunk 1]
```diff
@@ -236,6 +310,54 @@ async fn wait_for_job(
...
+            while let Some(status) = stream.try_next().await? {
+                debug!("pod watch event {:#?}", status);
+                match status {
+                    WatchEvent::Added(pod) | WatchEvent::Modified(pod) => {
+                        trace!("found pod: {}", pod.name_unchecked());
+                        debug!("pod status: {:#?}", pod.status);
+
+                        if let Some(status) = &pod.status {
+                            if let Some(phase) = &status.phase {
+                                if phase == "Succeeded" {
```
Comment (dav1do) [inline hermetic/src/cli/tester.rs:337]: Well what the heck... the only comment that was at all interesting (or pedantic?) disappeared... so submitting it again.

Nothing to change, but as an FYI you can do some fancy stuff with `match`. I find myself considering it when I have nested `if let` statements. For example,

```rust

match pod.status.and_then(|s| s.phase).as_deref() {
    Some("Suceeded") => {
        // woo
    }
    Some("Failed") => {
        //boo
    }
    Some(_) => {
        unreachable!();
    }
    None => {}
};
```

You can also use [conditionals](https://doc.rust-lang.org/book/ch18-03-pattern-syntax.html#extra-conditionals-with-match-guards) like `Some(v) if v == "Succeeded"` but I personally don't use those a ton.

File: simulations/basic-simulation.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+apiVersion: "keramik.3box.io/v1alpha1"
+kind: Simulation
+metadata:
+  name: recon
+spec:
+  scenario: recon-event-sync # or recon-event-key-sync if you're using rust-ceramic without recon data
+  throttleRequests: 280 # include to define a "steady sync" scenario where the worker creates about this many events/second
+  successRequestTarget: 10 # override the default value of 300 to determine how many events must be synced per second to other peers in the network
+  users: 2 # increase to generate more events, 1 is probably sufficient for our 300/s goal
+  runTime: 2
```
[the remark this answers is not in the document]
Comment (stephhuynh18) [reply simulations/basic-simulation.yaml:10]: Your exactly right. 2 minutes is to ensure things are going smoothly and i was going to make a follow up PR to adjust to 10

**Pull request merged**